app/providers/mi_appliance_type_pdp.rb

```diff
@@ -78,6 +78,17 @@ def filter(ats, mode = nil)
 
   private
 
+  def mi_url
+    project = if @current_user.project.blank?
+                Air.config.vph.default_project
+              else
+                @current_user.project
+              end
+
```

Comment:
Nop, since `@current_user.project` can be an empty string and then default project should be used.